cdc/metrics/grafana/tikv-cdc.json

```diff
@@ -2965,7 +2965,7 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "sum(rate(tikv_cdc_processor_table_memory_consumption_sum{tidb_cluster=\"$tidb_cluster\", capture=~\"$capture\"}[30s]) / rate(tikv_cdc_processor_table_memory_consumption_count{tidb_cluster=\"$tidb_cluster\", capture=~\"$capture\"}[30s])) by (capture)",
+              "expr": "sum(rate(tikv_cdc_processor_changefeed_memory_consumption_sum{tidb_cluster=\"$tidb_cluster\", capture=~\"$capture\"}[30s]) / rate(tikv_cdc_processor_changefeed_memory_consumption_count{tidb_cluster=\"$tidb_cluster\", capture=~\"$capture\"}[30s])) by (capture)",
```

Comment:
Please also paste the screenshot of these two metrics.

Reply:
pasted, PTAL -> https://github.com/tikv/migration/pull/242#issuecomment-1272883940